flask_server.py

```diff
@@ -60,7 +60,7 @@ def _v1_spec():
                 'url': 'http://www.apache.org/licenses/LICENSE-2.0.html'
             }
         },
-        'host': 'syntaxnet.askplatyp.us',
+        'host': '127.0.0.1:7000',
```

Comment:
We should find a way to get the current URL. Here it'll break the "production" version of this service at http://syntaxnet.askplatyp.us

Should probably be done in an other pull request